common/utils.js

```diff
@@ -466,6 +466,11 @@
             return result;
         }
 
+        function isBrowserIE() {
+            //Internet Explorer 6-11
+            return /*@cc_on!@*/false || !!document.documentMode;
```

Comment:
This works, as I have tested it in IE. You can try the url in IE and validate it.